config/gasCost.go

```diff
@@ -68,6 +69,25 @@ type ElrondAPICost struct {
 	DeleteFromReturnData uint64
 }
 
+// DynamicStorageLoad holds the signed coefficients of the func that will compute the gas cost
+// based on the trie depth.
+type DynamicStorageLoad struct {
+	A int64
+	B int64
+	C int64
```

Comment:
(This is optional)

The new code in `storageContext` suggests these three are coefficients of a quadratic polynomial in the trie depth.

It might be late to change this `struct` now, but instead of `A`, `B` and `C` they could be more precisely named:
* A → QuadraticCoefficient
* B → LinearCoefficient
* C → ConstantCoefficient

Or rename the `struct` to `DynamicStorageLoadCostCoefficients` and `A, B, C` to just `Quadratic, Linear, Constant`.

Reply:
Renamed.